src/mudlet.cpp

```diff
@@ -3062,18 +3062,14 @@ void mudlet::toggleMuteForProtocol(bool state, QAction* toolbarAction, QAction*
     // Notify when all media is muted or all media is unmuted. Helps if the shortcut is hit accidentally.
     if (isMediaMuted || mediaUnmuted()) {
         for (auto pHost : mHostManager) {
-            if (isMediaMuted) {
-#if defined(Q_OS_MACOS)
-                pHost->postMessage(tr("[ INFO ]  - Mudlet is muted. Press COMMAND-K to unmute."));
-#else
-                pHost->postMessage(tr("[ INFO ]  - Mudlet is muted. Press ALT-K to unmute."));
-#endif
-            } else {
-#if defined(Q_OS_MACOS)
-                pHost->postMessage(tr("[ INFO ]  - Mudlet is unmuted. Press COMMAND-K to mute."));
-#else
-                pHost->postMessage(tr("[ INFO ]  - Mudlet is unmuted. Press ALT-K to mute."));
-#endif
+            QKeySequence* sequence = new QKeySequence(*pHost->profileShortcuts.value(qsl("Mute All Media")));
+ 
+            if (!sequence->toString().isEmpty()) {
```

Comment:
Just because there isn't a defined key sequence that is not a good reason to NOT show a message - you will have to give a shorter one without the shortcut!

Reply:
Good call.  Done.  Second set of messages was after clearing the shortcut.
<img width="802" alt="Screenshot 2023-09-04 at 8 40 35 PM" src="https://github.com/Mudlet/Mudlet/assets/3058178/31c12585-5c2e-41be-9cfd-4e456d63371f">